rasa_nlu/data_router.py

```diff
@@ -235,6 +235,8 @@ def parse(self, data):
         response, used_model = self.project_store[project].parse(data['text'],
                                                                  time,
                                                                  model)
+        response['project'] = project
```

Comment:
can we move that to https://github.com/RasaHQ/rasa_nlu/blob/master/rasa_nlu/project.py#L124 and from that function only return `response `? (because if we set the project on the response anyways, there is no need to return a tuple from the `project.parse`).

Reply:
But wouldn't the used_model be anyway required at line number 243?

Reply:
sure, but we can just use `response.get('model')` there.

Reply:
Yes.
I will make changes and push